bootstrap.sh

```
@@ -1,20 +1,34 @@
 #!/bin/bash
+
```

Add strict shell options and basic safety at the top of the script. The task requested robust guards (`set -euo pipefail`) so failures are caught early and the script doesn't continue on error. Also consider setting `IFS=$'\n\t'` to avoid word-splitting surprises. Example to add immediately after the shebang:

```bash
set -euo pipefail
IFS=$'\n\t'
```

This is a HIGH-severity fix because without it the script may silently continue after failures, making validation unreliable.